src/litserve/server.py

```diff
@@ -281,6 +282,8 @@ def __init__(
             raise ValueError("batch_timeout must be less than timeout")
         if max_batch_size <= 0:
             raise ValueError("max_batch_size must be greater than 0")
+        if isinstance(spec, OpenAISpec):
```

Comment:
we can't special case OpenAISpec here
if we pass a spec, we need to determine if the spec overrides attributes on the LitAPI

Reply:
Ok fair, let's keep things hard-coded in this PR and we'll address it later